.github/settings.yml

```diff
@@ -147,8 +169,8 @@ labels:
 - name: lifecycle/duplicate
   color: e5fefc
   description: Duplicate of another issue or PR
-
-- name: dev/help
+## The different states of an issue or pull request for easier collaboration
+- name: dev/help-wanted
```

Comment:
dev/help-wanted and dev/blocked sound incredibly similar, and even dev/second-opinion sounds too similar to be an individual label (especially since you will get my second opinion anyway if you request me to review).

Reply:
fair point - shall we remove all of them? Or keep one? Which?

Reply:
I would argue that if somebody needs help, he should directly address it to a team or team-member. Setting the label and waiting until somebody comes to support seems inefficient. Therefore, I would vote to remove them all.

Reply:
needs help is not necessarily blocked. I can still continue but it will take some more time to proceed. I can figure it out but would be faster with help. Or I don't know how to proceed and need help.

Blocked for me means it's depending on something that is outside of my control and not that I don't understand the thing I'm supposed to do. For me, blocked means, I'm waiting for something or someone that I cannot influence.